core/src/main/java/aima/core/search/basic/support/BidirectionalSearch.java

```diff
@@ -0,0 +1,306 @@
+package aima.core.search.basic.support;
+
```

Comment:
This is located in the wrong package, the description in the book is described in the context of uninformed search algorithms, and therefore the version included in the 'core' module should adhere to this restriction. A basic Bidrectional implementation should go in:

aima.core.search.basic.uninformed

However, more advanced versions could be added to the 'extra' module as alternative implementations. That either exhibit more functionality or some other plus over the basic description in the book. Note: I am under the impression that the Bidirectional search description will change in AIMA4e based on recent research on the approach (TBD).